gear-test/src/proc.rs

```diff
@@ -282,7 +282,7 @@ where
                 let actor = state.actors.get(&dispatch.message.dest()).cloned();
 
                 let journal = core_processor::process::<Ext, E>(
-                    actor,
+                    actor.unwrap_or_default(),
```

Comment:
we should panic before for that case. if we have message addressed to user in message (dispatch) queue - this is a bug

Reply:
done

Reply:
> we should panic before for that case. if we have message addressed to user in message (dispatch) queue - this is a bug

So the `unwrap_or_default` should be replaced with `expect`

Reply:
I accidentally undo it in rebase, fixed it